src/renderer/components/InlinePK.tsx

```diff
@@ -105,7 +105,7 @@ const InlinePK: React.FC<{
 
 InlinePK.defaultProps = {
   className: '',
-  formatLength: 32,
```

Comment:
please no, don't minimse by default - its a UX choice that should be only used where space is a constraint.